Jenkinsfile

```diff
@@ -1,56 +1,74 @@
 #!/usr/bin/groovy
```

Comment:
do you plan to move this to `.jenkins/`?

Reply:
i've gone ahead and moved this now and updated the build pipeline to look for it in the new location.